src/features/user/TreeMapper/Import/index.tsx

```diff
@@ -175,11 +175,7 @@ export default function ImportData(): ReactElement {
           </div>
         </div>
         <div className={styles.mapContainer}>
-          <MapComponent
-            geoJson={geoJson}
-            setGeoJson={setGeoJson}
-            setActiveMethod={setActiveMethod}
-          />
+          <MapComponent geoJson={geoJson} />
```

Comment:
Here there is an error stating `Type 'null' is not assignable to type 'GeoJSON'`. Could you add a condition to check if geoJson is not null?
` {geoJson && <MapComponent geoJson={geoJson} />}`

Reply:
I have not encountered such an error  for this file.